src/streams.h

```diff
@@ -526,14 +528,16 @@ class AutoFile
      */
     bool IsNull() const         { return (file == nullptr); }
 
+    /** Implementation detail, only used internally. */
+    std::size_t detail_fread(Span<std::byte> dst);
```

Comment:
fa2276355889ff7bcad95aa978863107c36a89f7

Minor: new function is called `detail_fread` but commit and commit message mention `detail_read`.